kubernetes/deployment-production.tmpl

```diff
@@ -96,6 +113,12 @@ spec:
             limits:
               memory: "275Mi"
               cpu: "500m"
+          livenessProbe:
+            tcpSocket:
+              port: 3000
```

Comment:
I assume we're switching to use the `/Index` path via HTTP instead of tcp?

Reply:
Yup, going to switch that over this afternoon.